guacamole/src/main/webapp/app/navigation/templates/guacMenu.html

```diff
@@ -0,0 +1,7 @@
+<div class="menu-dropdown" ng-class="{open: menuShown}" ng-click="toggleMenu()">
+    <div class="menu-title">{{menuTitle}}</div>
+    <div class="menu-indicator"></div>
+
+    <!-- Menu options -->
+    <div class="options" ng-transclude></div>
```

Comment:
Why is this called options? It's basically just the content of whatever you want to throw in there, right?